lib/public/views/Runs/Details/index.js

```diff
@@ -86,37 +82,42 @@ const runDetails = (model) => {
-                h('button.btn.btn-primary.w-15.mh2#create', {
-                    onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
-                }, 'Add Log Entries'),
-                h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
-            ]),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-success.w-15.mh2', {
-                    onclick: () => model.runs.updateRunTags(),
-                    title: 'This action will overwrite existing tags',
-                    disabled: !model.runs.selectedTagsChanged(),
-                }, 'Update Tags'),
-                h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
-                    'Add one or multiple tags to the run. ',
-                    '(CTRL + click to select multiple tags).',
+            h('.flex-row', [
+                h('.flex.w-100', [
+                    h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-primary.w-15.mh2#create', {
+                            onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
+                        }, 'Add Log Entries'),
+                        h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
+                    ]),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-success.w-15.mh2', {
+                            onclick: () => model.runs.updateRunTags(),
+                            title: 'This action will overwrite existing tags',
+                            disabled: !model.runs.selectedTagsChanged(),
+                        }, 'Update Tags'),
+                        h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
+                            'Add one or multiple tags to the run. ',
+                            '(CTRL + click to select multiple tags).',
+                        ]),
+                    ]),
+                    h('select#tags-control.form-control.w-30.mh2', {
+                        multiple: true,
+                        onchange: ({ target }) => model.runs.setSelectedTags(target.selectedOptions),
+                    }, !tagsAvailable ? h('option', {
+                        onclick: () => model.router.go('?page=tag-create'),
+                    }, h('a#tagCreateLink', {
+                    }, noTagsText)) :
+                        allTags.isSuccess() ? [
+                            ...allTags.payload.map((tag) => h('option', {
+                                value: tag.id,
+                                selected: selectedTags.includes(tag.id),
+                            }, tag.text)),
+                        ] : h('option', { disabled: true }, 'Loading tags...')),
                 ]),
+                h('.justify-content-end', [runDetail(model, data.payload)]),
```

Comment:
I cannot find this CSS class either in the WebUI or Bookkeeping CSS `justify-content-end`
If you would like to create one with certain properties, feel free to add it to `app.css`

If you would like to add certain CSS styles to this node only you can rewrite it as below:
```
h('.justify-content-end', {
     style: 'display:flex; justify-content: flex-end'
}, [
     runDetail(model, data.payload)
]),
```

Reply:
Left it as one node only